client/driver/docker.go

```diff
@@ -238,14 +238,14 @@ func (d *DockerDriver) createContainer(ctx *ExecContext, task *structs.Task, dri
 	hostConfig.NetworkMode = driverConfig.NetworkMode
 	if hostConfig.NetworkMode == "" {
 		// docker default
-		d.logger.Println("[INFO] driver.docker: networking mode not specified; defaulting to bridge")
+		d.logger.Println("[DEBUG] driver.docker: networking mode not specified; defaulting to bridge")
 		hostConfig.NetworkMode = "bridge"
 	}
 
 	// Setup port mapping and exposed ports
 	if len(task.Resources.Networks) == 0 {
 		d.logger.Println("[DEBUG] driver.docker: No network interfaces are available")
```

Comment:
No this is meant for debugging so you can see if your interfaces are not being detected, which is generally useful. The inner if is a specific error mode that can't proceed at all. If you hit the error every time I wouldn't need to log this; you would know right away.